.prettierignore

```diff
@@ -1 +1,6 @@
-.prettierrc
\ No newline at end of file
+.prettierrc
+.github/
+node_modules/
+public/
+*.md
+package*.json
```

Comment:
Good call modifying the prettier ignore.

Personally, I'm okay with also formatting markdown files. But I'm also okay not doing it.

Reply:
done